script.js

```diff
@@ -77,3 +77,107 @@ function myFunction(x) {
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
+  document.getElementById('tech2').innerHTML = showItemInfo.technologies.tech2;
+  document.getElementById('tech3').innerHTML = showItemInfo.technologies.tech3;
+  document.getElementById('description').innerText = showItemInfo.description;
+  document.getElementById('image').style.backgroundImage = `url('${showItemInfo.image}')`;
+  document.getElementById('liveButton').setAttribute('href', showItemInfo.liveVersion);
+  document.getElementById('liveSource').setAttribute('href', showItemInfo.liveSource);
+});
+showModal1.addEventListener('click', () => {
+  document.getElementById('modal').classList.add('active');
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
+  document.getElementById('tech2').innerHTML = showItemInfo.technologies.tech2;
+  document.getElementById('tech3').innerHTML = showItemInfo.technologies.tech3;
+  document.getElementById('description').innerText = showItemInfo.description;
+  document.getElementById('image').style.backgroundImage = `url('${showItemInfo.image}')`;
+  document.getElementById('liveButton').setAttribute('href', showItemInfo.liveVersion);
+  document.getElementById('liveSource').setAttribute('href', showItemInfo.liveSource);
+});
+showModal2.addEventListener('click', () => {
+  document.getElementById('modal').classList.add('active');
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
+  document.getElementById('tech2').innerHTML = showItemInfo.technologies.tech2;
+  document.getElementById('tech3').innerHTML = showItemInfo.technologies.tech3;
+  document.getElementById('description').innerText = showItemInfo.description;
+  document.getElementById('image').style.backgroundImage = `url('${showItemInfo.image}')`;
+  document.getElementById('liveButton').setAttribute('href', showItemInfo.liveVersion);
+  document.getElementById('liveSource').setAttribute('href', showItemInfo.liveSource);
+});
+showModal3.addEventListener('click', () => {
+  document.getElementById('modal').classList.add('active');
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
+  document.getElementById('tech2').innerHTML = showItemInfo.technologies.tech2;
+  document.getElementById('tech3').innerHTML = showItemInfo.technologies.tech3;
+  document.getElementById('description').innerText = showItemInfo.description;
+  document.getElementById('image').style.backgroundImage = `url('${showItemInfo.image}')`;
+  document.getElementById('liveButton').setAttribute('href', showItemInfo.liveVersion);
+  document.getElementById('liveSource').setAttribute('href', showItemInfo.liveSource);
```

Comment:
- [ ] please make sure that the icons are displayed and also  make sure the align the content as in the Figma design in order to match and don't forget the button effect on hover and pressed 
![image](https://user-images.githubusercontent.com/61790587/179960149-2d993952-624e-4771-8332-5f03c57088b1.png)